### PR TITLE
fixed missing metadata issues

### PR DESCRIPTION
**Pull request by beijbom**
Fixes https://github.com/beijbom/coralnet/issues/282

### CODE REVIEW

File: project/visualization/utils.py [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+from PIL import ImageFile
 from django.conf import settings
 from django.core.files.storage import get_storage_class
 from django.db.models import Q
-import django.db.models.fields as model_fields
-from PIL import Image as PILImage
+
 from images.models import Point, Image, Metadata
 
+ImageFile.LOAD_TRUNCATED_IMAGES = True
```
Comment (beijbom) [inline project/visualization/utils.py:17]: @StephenChan : I added this here, but perhaps better to move to project level config? (`config/settings/base.py`)?

**Comment by beijbom** [comment]
@StephenChan : I got tired of seeing these error messages to figured I'd fix it. Was easier than I expected although can't be sure works for all images.

File: project/visualization/tests/test_utils.py [hunk 2]
```diff
@@ -1,16 +1,18 @@
 from __future__ import unicode_literals
+
+import os
 from io import BytesIO
+
 import mock
 from PIL import Image as PILImage
 from PIL.Image import SAVE as PIL_SAVE
-
+from django.conf import settings
```
Comment (beijbom) [inline project/visualization/tests/test_utils.py:9]: These import changes are due to using the pycharm auto-format imports.

Comment (StephenChan) [reply project/visualization/tests/test_utils.py:9]: Works for me.

File: project/visualization/utils.py (hunk 1 above)
Comment (StephenChan) [reply project/visualization/utils.py:17]: Moving to `config/settings/base.py` seems better, yeah. I just tried that and it works, so I'll add that change to this PR.

**Comment by StephenChan** [comment]
Thanks @beijbom , that's a nice find. I'll just push a commit to move the LOAD_TRUNCATED_IMAGES line to settings, as mentioned in my inline comment.

**Pull request merged**